src/main/java/me/zeroeightsix/kami/util/astaralg/Astarpathfinder.java

```diff
@@ -219,7 +220,7 @@ private static void pathfind() {
                 stuckPos = getPlayerBlockpos();
                 state = State.LAND_SAVE;
                 nopathfound = 0;
-                currentEvent.cancel();
+                elytraFlight.controlMode(currentEvent);
```

Comment:
i think this was the problem , i canceledone packet before starting the autoland

Reply:
that didn't fix it completely. still happens sometimes if you fly directly under a block but a lot less often but still. I am thinking about making a detection system for this issue